apps/dotcms-ui-e2e/src/support/shared/Navigation.ts

```diff
@@ -0,0 +1,24 @@
+class Navigation {
+    static visit(url: string, options?: Cypress.VisitOptions) {
+        cy.visit(url, options);
+    }
+
+    static assertPageUrlIs(url: string) {
```

Comment:
Why not use something like `cy.url().should('include','/dotAdmin/#/c/embedded-dashboard')`

Reply:
maybe, can't remember why I did that way.

will test your suggestion!